corehq/apps/zapier/signals/receivers.py

```diff
@@ -0,0 +1,13 @@
+from django.dispatch import receiver
+
+from corehq.apps.zapier.tasks import send_to_subscribers_task
+from corehq.toggles import ZAPIER_INTEGRATION
+from couchforms.signals import successful_form_received
+
+
+@receiver(successful_form_received)
+def send_form_subscribers(sender, xform, *args, **kwargs):
+    domain = xform.domain
+    if not ZAPIER_INTEGRATION.enabled(domain):
+        return
+    send_to_subscribers_task.delay(domain, xform)
```

Comment:
good point, i think it depends on how big we expect the object to be. it's also sometimes preferable to pass the object rather than the id in cases where stale objects are fine so that we avoid the hit to the db, so i suppose it's a trade-off. i'd say as long as we don't think the xforms passed here will be huge, then fine to leave as is.